docs/howto/static-files/index.txt

```diff
@@ -137,6 +137,22 @@ this by adding the following snippet to your urls.py::
     the given prefix is local (e.g. ``/media/``) and not a URL (e.g.
     ``http://media.example.com/``).
 
+Customizing the ignored pattern list
+====================================
+
+When collecting files through the :djadmin:`collectstatic` management command,
+files matching the pattern list ``['CVS', '.*', '*~']`` are ignored by default.
+If you need to customize this list on a more stable way than providing the
+``--ignore`` command option at each invocation, you can provide a custom
+:class:`~django.apps.AppConfig` class, override the ``ignore_patterns``
+attribute of this class and specify that class path inside your
+:setting:`INSTALLED_APPS` setting::
+
+    from django.contrib.staticfiles.apps import StaticFilesConfig
+
+    class IgnorePatternsAppConfig(StaticFilesConfig):
```

Comment:
I'd try for a more generic name, e.g. `MyStaticFilesConfig` (though I don't care for the "My" convention). If we add more settings this way, naming the class based on the options doesn't make sense.